MAUI/Popup/migration.md

```diff
@@ -0,0 +1,215 @@
+
+To migrate easier from Xamarin.Forms SfPopupLayout to .NET MAUI Popup, we kept most of the APIs from Xamarin SfPopupLayout in MAUI SfPopup. However, to maintain the consistency of API naming in MAUI SfPopup, we renamed some of the APIs. The APIs that have been changed in MAUI SfPopup from Xamarin SfPopupLayout are detailed as follows.
+
+## Namespaces 
+
+<table>
+<tr>
+<th>Xamarin SfPopupLayout </th>
+<th>.NET MAUI SfPopup</th></tr>
+<tr>
+<td>Syncfusion.XForms.SfPopupLayout</td>
+<td>Syncfusion.Maui.Popup</td></tr>
+</table>
+
+## Initialize Control
+
+To initialize the control, import the popup namespace and initialize SfPopup as shown in the following code sample.
+
+<table>
+<tr>
+<th>Xamarin SfPopupLayout</th>
+<th>.NET MAUI SfPopup</th></tr>
+<tr>
+<td>
+{% tabs %}
+{% highlight xaml tabtitle="MainPage.xaml" %}
+
+<ContentPage xmlns:sfPopup="clr-namespace:Syncfusion.XForms.PopupLayout;assembly=Syncfusion.SfPopupLayout.XForms">
+<sfPopup:SfPopupLayout x:Name="popupLayout">
+    <sfPopup:SfPopupLayout.Content>
+        <Button x:Name="clickToShowPopup" Text="ClickToShowPopup" 
+        VerticalOptions="Start" HorizontalOptions="FillAndExpand"
+        Clicked="ClickToShowPopup_Clicked"/>
+    </sfPopup:SfPopupLayout.Content>
+</sfPopup:SfPopupLayout>
+</ContentPage>
+
+{% endhighlight %}
+{% highlight c# tabtitle="MainPage.xaml.cs" %}
+
+public partial class MainPage : ContentPage
+{
+    public MainPage()
+    {
+        InitializeComponent();
+    }
+    private void ClickToShowPopup_Clicked(object sender, EventArgs e)
+    {
+        popupLayout.Show();      
+    }
+}
+
+{% endhighlight %}
+{% endtabs %}
+</td>
+<td>
+{% tabs %}
+{% highlight xaml tabtitle="MainPage.xaml" %}
+
+<ContentPage xmlns:syncfusion="clr-namespace:Syncfusion.Maui.Popup;assembly=Syncfusion.Maui.Popup">            
+    <StackLayout>
+            <Button x:Name="clickToShowPopup" Text="ClickToShowPopup" 
+        VerticalOptions="Start" HorizontalOptions="FillAndExpand"
+        Clicked="ClickToShowPopup_Clicked"/>
+            <syncfusion:SfPopup x:Name="popup"/>
+        </StackLayout>
+</ContentPage>
+
+{% endhighlight %}
+{% highlight c# tabtitle="MainPage.xaml.cs" %}
+public partial class MainPage : ContentPage
+{
+    public MainPage()
+    {
+        InitializeComponent();
+    }
+
+    private void ClickToShowPopup_Clicked(object sender, EventArgs e)
+    {
+        popup.Show();
+    }
+}
+
+{% endhighlight %}
+{% endtabs %}
+</td></tr>
+</table>
+
+## Classes
+
+<table> 
+<tr>
+<th>Xamarin SfPopupLayout</th>
+<th>.NET MAUI SfPopup</th>
+<th>Description</th></tr>
+<td>SfPopupLayout</td>
+<td>Sfpopup</td>
+<td>Displays an alert message with customizable buttons or loads a desired view within a pop-up window.</td></tr>
+<tr>
+<td>PopupView</td>
+<td>Nil</td>
+<td>Not supported</td></tr>
+</table>
+
+## Properties
+
+N> All the properties from Xamarin.Forms PopupView class are moved to MAUI SfPopup class itself.
```

Comment:
N> All the properties from the Xamarin.Forms PopupView class is moved to the MAUI SfPopup class itself.